docs/style-guide.rst

```diff
@@ -888,13 +888,19 @@ Function Names
 Functions other than constructors should use mixedCase. Examples: ``getBalance``, ``transfer``, ``verifyOwner``, ``addMember``, ``changeOwner``.
 
 
+Internal Function Names
+=======================
+
+Internal functions which have the ``internal`` or ``private`` modifiers should use mixedCase with leading underscore. Examples: ``_processTransaction``, ``_addMember``, ``_modifyConfig``.
```

Comment:
I think this requires a bit more discussion between stakeholders. @bit-shift suggested also that if the guide is closely followed then the visibility specifier is right below the function name and that should already show it is internal or private.

This also a bit similar to Javascript's (even ES6) workaround to not having private members (they prefix functions as a convention).